moulinette/actionsmap.py

```diff
@@ -368,17 +368,23 @@ def __init__(self, parser_class, namespaces=[], use_cache=True,
         for n in namespaces:
             logger.debug("loading actions map namespace '%s'", n)
 
-            if use_cache:
+            actionsmap_yml = '%s/actionsmap/%s.yml' % (pkg.datadir, n)
+            actionsmap_pkl = '%s/actionsmap/%s-%s.pkl' % (pkg.cachedir, n, os.stat(actionsmap_yml).st_size)
```

Comment:
Is relaying on actionmap file size something strength enough for determining its modification? Should we not use `st_mtime` - the time of most recent content modification - instead?